ohw22/satellites.md

```diff
@@ -64,4 +66,4 @@ For help with timezone conversions, see [https://www.timeanddate.com](https://ww
 :hidden:
 
 maine/index
```

Comment:
This should add the satellite to the menu in the sidebar, and get the menu to show up on it's page too.

```suggestion
maine/index
seattle/index
```

Reply:
That's what we want. I'll commit your suggestion

Reply:
Awesome, thanks!